egov-indexer/open-artifact-indexer.yml

```
@@ -0,0 +1,104 @@
+  mappings:
+    - topic: open-artifact-topic
+      configKey: INDEX
+      indexes:
+        - name: open-artifact-index
+          type: general
+          id: $.artifactNumber
+          isBulk: false
+          timeStampField: $.createdDate
+          jsonPath: $
+          customJsonMapping:
+            indexMapping: { "Data": { "artifactDetails": { } } }
```

YAMLlint (1.37.1)

[error] too many spaces inside braces

(braces)